thorlcr/activities/loop/thloopslave.cpp

```diff
@@ -439,6 +438,7 @@ class CLoopSlaveActivity : public CLoopSlaveActivityBase
     }
     CATCH_NEXTROW()
     {
+        ActivityTimer t(slaveTimerStats, timeActivities);
```

Comment:
I think this change looks correct in principle, but in practice, unless the lookahead time that is being performed by the feeder, the timing calculation may well be more inaccurate.

Reply:
I was looking at dict15 from the performance suite.  The loop's total time was greater than the upstream activities total time.  Moving the loop activity's time tracking from CNextRowFeeder to nextRow fixed the issue.  Why is this wrong?